### PR TITLE
Syntax warning equality

### PR DESCRIPTION
**Pull request by jrisebor**
When running `okta-awscli` for the first time on Python 3.8+ it throws a warning like this:
```
/usr/local/Cellar/okta-awscli/0.4.4/libexec/lib/python3.8/site-packages/oktaawscli/okta_auth_mfa_app.py:102: SyntaxWarning: "is not" with a literal. Did you mean "!="?
  if factor_name is not "":
```
After doing a quick bit of research I found a [blog post](https://adamj.eu/tech/2020/01/21/why-does-python-3-8-syntaxwarning-for-is-literal/) that explains the problem pretty clearly. I also found that [empty strings are considered `False`](https://docs.python.org/3/library/stdtypes.html#truth-value-testing), so I was able to simplify the comparison since `_build_factor_name` appears to only return strings.

I'm not entirely sure this codepath is testable/reachable though, since there doesn't seem to be a case where `_build_factor_name` returns an empty string.

### CODE REVIEW

File: oktaawscli/okta_auth_mfa_app.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def _choose_factor(self, factors):
             # print out the factors and let the user select
             for i, factor in enumerate(factors):
                 factor_name = self._build_factor_name(factor)
-                if factor_name is not "":
+                if not factor_name:
```
Comment (jmhale) [inline oktaawscli/okta_auth_mfa_app.py:102]: The warning on the identity check makes sense. 

However, this line is checking that `factor_name` is not empty, so this change is causing the factors to not get printed out:

```
Multi-factor Authentication required for application.
Pick a factor:
Selection:
```

So I'd do this instead:
```suggestion
                if factor_name:
```

Which should restore the factor list, while still using an equality check.
```
Multi-factor Authentication required for application.
Pick a factor:
[ 0 ] token:software:totp( GOOGLE ) : 
[ 1 ] Unknown MFA type: u2f
[ 2 ] token:software:totp( OKTA ) : 
Selection:
```

**Review by jmhale — CHANGES_REQUESTED** [review]

File: oktaawscli/okta_auth_mfa_app.py (hunk 1 above)
Comment (jrisebor) [reply oktaawscli/okta_auth_mfa_app.py]: Ah yeah, that makes sense. Updated!

Comment (jmhale) [reply oktaawscli/okta_auth_mfa_app.py]: Awesome! Thanks!

**Review by jmhale — APPROVED** [review]
LGTM

**Pull request merged**